### DISCUSSION TITLE
Can’t install BHTikTok

### DISCUSSION THREAD

**Discussion by 8bitkitten** [dayanch96/BHTikTok-Plus#18] [Q&A] [answered]
Hello, every time I try to sign the app I get this error ![image](https://github.com/dayanch96/BHTikTok-Plus/assets/37677485/36f09a99-944f-476c-82d1-be56b7ce0714) I've tried multiple versions including the latest one but none seem to work. I can sign other apps just fine. Is there a way to fix this?

**Comment by dayanch96** [comment, marked as answer]
Hey. Try signing using Sideloadly

**Reply by 8bitkitten** [reply]
That worked thanks
